### DISCUSSION TITLE
How to add a simple field in the checkout page?

### DISCUSSION THREAD

**Discussion by Ektarish** [woocommerce/woocommerce#44140] [Checkout Flow]
I installed a fresh version of WooCommerce the other day, and now i'm trying to add a custom field in the checkout as in Italy we need to collect the TAX ID of people to emit invoices even for B2C, how can i add a new field in the Woo Block in the checkout?

Thanks!

**Comment by opr** [comment]
Hi @Ektarish we are working on this now! [Please see our latest update](https://developer.woo.com/2024/01/26/woocommerce-blocks-extensibility-snaps-additional-checkout-fields-progress/)

In the meantime if you really need this then you can create an inner block for the checkout, [this tutorial might help](https://developer.woo.com/2023/08/07/extending-the-woocommerce-checkout-block-to-add-custom-shipping-options/)
